bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/dto/NAWelcome.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.netatmo.internal.api.dto;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.netatmo.internal.api.data.NetatmoConstants.PresenceLightMode;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.types.State;
+import org.openhab.core.types.UnDefType;
+
+/**
+ * The {@link NAWelcome} holds status informations of a camera.
+ *
+ * @author Gaël L'hopital - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class NAWelcome extends NAModule {
+    private @Nullable OnOffType status;
+    private @Nullable String vpnUrl;
+    private boolean isLocal;
+    private @Nullable OnOffType sdStatus;
+    private @Nullable OnOffType alimStatus;
+    private PresenceLightMode lightModeStatus = PresenceLightMode.UNKNOWN;
+
+    /**
+     * If camera is monitoring (on/off)
+     *
+     * @return status
+     **/
+    public State getStatus() {
+        OnOffType localStatus = status;
+        return localStatus != null ? localStatus : UnDefType.NULL;
```

Comment:
Should we use UnDefType.NULL or UnDefType.UNDEF ?
Same remark for all other places.

Reply:
That is less clear for me after I read a recent comment from Kai. I will have to find again his comment.

Reply:
Yes, this is always a fight in my mind also. I had the idea that UNDEF was reserved for channels that had not been initialized

Reply:
No, this is NULL for channels unintialized.